packages/flutter/lib/src/rendering/debug.dart

```diff
@@ -5,7 +5,7 @@
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 
-export 'package:flutter/services.dart' show debugPrint;
+export 'package:flutter/services.dart';
```

Comment:
Oops, yeah. This symbol no longer exists in `flutter/services.dart` and the latest analyzer complains about that for exports. I should have removed the whole line.